### PR TITLE
Add timeout error message to shadow reset

### PR DESCRIPTION
**Pull request by Wiljami**
The user is now informed when the ajax call times out.

When the ajax returns 504 timeout message the user now gets an error message about it.

Unfortunately there is no easy way to know if the shadow is actually resetted or not.

Related #315, #187

TODO: translations once code is ok

### CODE REVIEW

**Comment by Wiljami** [comment]
Admittedly it is misleading. Should the message then say that the reset might or might not have occurred? As far as I know there is no way to check when the shadow has been reset last time.

**Comment by elguitar** [comment]
Maybe the text could say that the reset is running in the background and that the user should check the shadow in a while. It should also say that if the user runs into trouble, she should find help from the docs and provide a link there.

**Comment by Wiljami** [comment]
Changed the text and also changed the background color of the box to yellow to indicate that this is not outright error.

Screenshot from 2020-08-03 14-20-44

File: modules/sitestatus.php [hunk 1]
```diff
@@ -410,6 +410,7 @@ public static function seravo_shadows_postbox() {
             </div>
           </div>
           <div class="alert" id="alert-failure"><button class="closebtn">&times;</button><p><?php _e('Failure!', 'seravo'); ?></p></div>
+          <div class="alert" id="alert-timeout"><button class="closebtn">&times;</button><p><?php _e('The shadow reset is still running on the background. You should check the status of the shadow after few minutes. If there are problems with the shadow, see the documentation from the link above.', 'seravo'); ?></p></div>
```
Comment (elguitar) [inline modules/sitestatus.php:413]: after **a** few

File: languages/seravo-fi.po [hunk 2]
```diff
@@ -741,6 +741,16 @@ msgstr ""
...
+#: modules/plugin-checker.php:40
+#, fuzzy
+#| msgid "Unnecessary plugins"
+msgid "Bad plugins"
+msgstr "Tarpeettomat lisäosat"
+
+#: modules/plugin-checker.php:49
+msgid "Bad plugins found: "
+msgstr ""
+
```
Comment (elguitar) [inline languages/seravo-fi.po:753]: Untranslated

Comment (elguitar) [reply languages/seravo-fi.po:753]: But this has nothing to do with this PR :D Resolving

**Pull request merged**